typescript/types.ts

```diff
@@ -192,8 +192,12 @@ export type ExecuteResult = {
   /**
    * The result of executing the prompt.
    */
-  data: JSONValue;
-
+  data:
+    | JSONValue
+    | {
+        kind: "string" | "file_uri" | "base64";
+        value: string;
+      };
```

Comment:
Ah, right, I guess we just rely on the mime_type below to know the represented type